### DISCUSSION TITLE
Admin API support in cerbos-sdk-java

### DISCUSSION THREAD

**Discussion by bbodensieck** [cerbos/cerbos#1607] [Q&A] [answered]
Hi folks,
I just saw that you support the Admin API in the JS SDK.
https://community.cerbos.dev/t/2234213/quick-question-about-the-cerbos-node-sdk-there-isn-t-an-exis#f521c36a-c9f1-49ef-ae56-dcb0e5999c4e

Do you plan to support the Admin API in cerbos-sdk-java as well?

Have a nice day,
Björn

**Comment by oguzhand95** [comment, marked as answer]
Hey @bbodensieck,

Yes, adding the Admin API support is planned. I will communicate your interest with the team.

Have a nice day!
